charts/redis-ha/templates/redis-tls-secret.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if and .Values.redis.tlsPort (not .Values.tls.secretName) -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  metadata:
+  name: {{ template "redis-ha.fullname" . }}-tls-secret
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "labels.standard" . | indent 4 }}
+type: Opaque
+data:
+  {{- if .Values.tls.caCertFile }}
+  {{ .Values.tls.caCertFile }}: {{ .Files.Get "certs/ca.crt" | b64enc }}
+  {{- end }}
+  {{- if .Values.tls.certFile }}
+  {{ .Values.tls.certFile }}: {{ .Files.Get "certs/redis.crt" | b64enc }}
+  {{- end }}
+  {{- if .Values.tls.keyFile }}
+  {{ .Values.tls.keyFile }}: {{ .Files.Get "certs/redis.key" | b64enc }}
+  {{- end }}
```

Comment:
Aren't these required if TLS is enabled?

Reply:
Do you mean the ca.crt, redis.crt and redis.key files in the cert directory? 
Yes, they are required when TLS is enabled and only when user doesn't provide their own secret through 'tls.secretName'
I haven't included the certificates because my guess is that users will provide their own

Reply:
@SarjakSShah My issue has more to do with us not checking is TLS is enabled first. I understand what you're doing with this, but I think we also need to check if TLS is enabled, just incase someone adds in their certs, but doesn't enable TLS.

Reply:
So I already check if redis.tlsPort is defined 
If someone defines a tlsPort then that means they need these certs